source/_integrations/alexa.smart_home.markdown

```
@@ -157,7 +157,7 @@ Next you need create a Lambda function.
 
 - Click `Functions` in the left navigation bar, to display the list of your Lambda functions.
 - Click `Create function`, select `Author from scratch`, then input a `Function name`.
-- Select *Python 3.9*, *Python 3.8* or *Python 3.7* as `Runtime`.
+- Select *Python 3.* as `Runtime`.
```

markdownlint-cli2 (0.17.2)

Emphasis style
Expected: underscore; Actual: asterisk

(MD049, emphasis-style)